### DISCUSSION TITLE
Mail app keeps freezing and crashing when trying to link personal gmail account

### DISCUSSION THREAD

**Discussion by realseamusdykmg** [elementary/triage#258] [Q&A]
Hi. Whenever I try to add my personal Gmail account, there seems to be an issue of Mail freezing and then crashing. It's been a problem for a while, but I didn't get around to asking until now. Help please!

**Comment by ghost** [comment]
Hello budy, how are you?  I hope so.  One solution for you to add your Google account in elementary OS is to install the gnome account manager, so you will sync the mail, calendar, tasks and drive applications in the elementary file manager.  This is the solution I found to use Google services in elementary.  To install open the terminal and type the command `sudo apt install gnome-online-accounts` after installation run the command `gnome-control-center online-accounts
 ` and enter your Google credentials and once completed just close the app and your account is already working on elementary apps. Espero ter ajudado :)

**Comment by realseamusdykmg** [comment]
Hey there,

Thanks for the tip on how to add my Google account to elementary OS. I
followed your instructions and it worked perfectly! However, I have a quick
question. Should it be opening a copy of the Ubuntu settings app when I run
the command "gnome-control-center online-accounts"? Just wanted to make
sure I did everything correctly.

Thanks again for your help!

Best regards,
Seamus

*This message was written by GrammarlyGO AI*

On Fri, Jun 23, 2023 at 11:57 AM Tiago Castro ***@***.***>
wrote:

> Hello budy, how are you? I hope so. One solution for you to add your
> Google account in elementary OS is to install the gnome account manager, so
> you will sync the mail, calendar, tasks and drive applications in the
> elementary file manager. This is the solution I found to use Google
> services in elementary. To install open the terminal and type the command sudo
> apt install gnome-online-accounts after installation run the command gnome-control-center
> online-accounts and enter your Google credentials and once completed just
> close the app and your account is already working on elementary apps.
> Espero ter ajudado :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/elementary/discussions/258#discussioncomment-6264331>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6QEC7ZHKIOHBZEOZU6E4HLXMXDHLANCNFSM6AAAAAAZPBK2KY>
> .
> You are receiving this because you authored the thread.
>
--

**Reply by ghost** [reply]
Hi, I'm glad the tip worked.  Answering your question, yes, when opening the command "gnome-control-center online-accounts" it opens the gnome configuration panel, because the online accounts service is a complement to the gnome settings, so for it to work it needs the complete panel ( since gnome has by default the option to add your accounts).  It's a way I found to add my Google email, drive, tasks and calendar on elementary OS.  So every time you want to add or remove account run the command "gnome-control-center online-accounts".  I'm studying elementary os source code in order to create a similar tool.
